File: src/main/java/com/mynimef/bot/callback/ICallback.java
package com.mynimef.bot.callback;

import com.mynimef.bot.containers.IButtonsContainer;
import com.mynimef.bot.containers.IFileAdd;
import com.mynimef.bot.containers.IFileContainer;

public interface ICallback extends IFileContainer, IButtonsContainer, IFileAdd<ICallback> {
}

File: src/main/kotlin/com/mynimef/bot/actions/ICallback.kt
package com.mynimef.bot.actions

import com.mynimef.bot.IBot


interface ICallback {

    fun callback(
        chatId: String?,
        messageId: Int?,
        message: String?,
        username: String?,
        firstName: String?,
        lastName: String?,
        bot: IBot?
    )

}
